### PR TITLE
Chore/semantic release

### PR DESCRIPTION
**Pull request by nellyk**
adding semantic-release to automate releases

### CODE REVIEW

**Review by mtomcal — DISMISSED** [review]

**Review by JAdshead — DISMISSED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -27,22 +27,13 @@
         "@babel/traverse": "^7.8.4",
         "@babel/types": "^7.8.3",
         "convert-source-map": "^1.7.0",
-        "debug": "^4.1.0",
+        "json5": "^2.1.0",
```
Comment (JamesSingleton) [inline package-lock.json:32]: Why did this go from `debug` to `json5`?

**Comment by JamesSingleton** [comment]
@nellyk,

There shouldn't be any changes in the `package-lock.json` right?

File: package-lock.json (hunk 1 above)
Comment (nellyk) [reply package-lock.json:32]: thanks for catching this i've updated this

**Comment by JamesSingleton** [comment]
@nellyk, some of the plugins for semantic release are not installed as devDeps.

**Pull request closed without merging**